fuseki/sample-ubd.trig

```diff
@@ -46,16 +46,16 @@ graph <cube-project/ubd/dataset> {
 
 <cube-project/ubd/dimensions-metadata> void:inDataset <ubd-example> .
 graph <cube-project/ubd/dimensions-metadata> {
-  <cube-project/ubd/dimensions-metadata> a cc:DimensionsMetadataCollection ;
-    schema:hasPart <cube-project/ubd/dimensions-metadata?dimension=https://environment.ld.admin.ch/foen/ubd/28/dimension%20year> ;
+  <cube-project/ubd/dimensions-metadata> a cc:DimensionMetadataCollection , hydra:Resource ;
+    schema:hasPart <cube-project/ubd/dimensions-metadata?dimension=https://environment.ld.admin.ch/foen/ubd/28/dimension-year> ;
     schema:hasPart <cube-project/ubd/dimensions-metadata?dimension=https://environment.ld.admin.ch/foen/ubd/28/station> ;
     schema:hasPart <cube-project/ubd/dimensions-metadata?dimension=https://environment.ld.admin.ch/foen/ubd/28/aggregation> ;
     schema:hasPart <cube-project/ubd/dimensions-metadata?dimension=https://environment.ld.admin.ch/foen/ubd/28/pollutant> ;
     schema:hasPart <cube-project/ubd/dimensions-metadata?dimension=https://environment.ld.admin.ch/foen/ubd/28/unit-id> ;
   .
 
-  <cube-project/ubd/dimensions-metadata?dimension=https://environment.ld.admin.ch/foen/ubd/28/shape/dimension%20year>
-    schema:about <https://environment.ld.admin.ch/foen/ubd/28/dimension%20year> ;
+  <cube-project/ubd/dimensions-metadata?dimension=https://environment.ld.admin.ch/foen/ubd/28/dimension-year>
+    schema:about <https://environment.ld.admin.ch/foen/ubd/28/dimension-year> ;
```

Comment:
Unfortunately (RDF/)JS is rather bad at comparing same URIs with mixed up escaping. We will have to avoid escaped characters in the property identifiers

Reply:
I think that makes sense anyway. We just have to make sure we handle auto-generated properties correctly, but I doubt users need escaped characters in properties they choose manually.